roles/_common/tasks/install.yml

```
@@ -37,7 +37,7 @@
     system: true
     shell: "/usr/sbin/nologin"
     group: "{{ _common_system_group }}"
-    home: "{{ _common_config_dir | default('/') }}"
+    home: "{{ _common_config_dir | default('/') | ternary(_common_config_dir, '/') }}"
```

Using both the default filter and ternary at the same time looks a bit strange to me. IMHO passing the `true` parameter to the default filter would be the appropriate fix.

```suggestion
    home: "{{ _common_config_dir | default('/', true) }}"
```